### ISSUE TITLE
Aerich puts all migrations in one app, I expect per-app migrations

### ISSUE THREAD

**Issue by syabro** [tortoise/aerich#171]
I have config

```
TORTOISE_ORM = {
    "connections": {"default": DATABASE_URI},
    "apps": {
        "thirdparty": {'models': ['aerich.models']},
        "zendr": {
            "models": [
                "subscriptions.models",
                "users.models",
                "userbots.models",
            ],
            "default_connection": "default",
        },
    },
}
```

When I run `init-db` I got

```
(zendr) ▲ zendr aerich init-db
Success create app migrate location .migrations/thirdparty
Success generate schema for app "thirdparty"

(zendr) ▲ zendr ls .migrations/
thirdparty
```

Zendr app models were put into thirdparty directory. I expect it to have `thirdparty` and `zendr` folders.

What do I do wrong?

**Comment by long2ice** [comment]
If you use more than one app, use `--app` option, or just use one

**Comment by syabro** [comment]
@long2ice emm, if I have 10 apps, I have to run command 10 times?

**Comment by long2ice** [comment]
Usually one app one database

**Comment by syabro** [comment]
Why call it `apps` then if you're emulating Django interface when `app` has very different meaning?

**Comment by abe-winter** [comment]
yeah this bit me as well -- `init-db` created an initial migration for all apps, but then `migrate` is only migrating the first app

(per this https://github.com/tortoise/aerich/blob/af632218750c1b6ee8571f70bd5de3d1eec3d2d2/aerich/cli.py#L76)

is it worth creating an `--all-apps` flag for the migrate command? I'm happy to submit a PR if that's useful

**Comment by abe-winter** [comment]
I did a POC here with an `--all-apps` flag https://github.com/abe-winter/aerich

It lets the `init-db` and `migrate` commands operate on multiple tortoise apps at once by supporting `app=None` in a bunch of places

my use case for apps is to let a fastapi project import plugins which have sql tables that get included in migrations -- I may be misusing / misunderstanding what tortoise apps are for, sorry in advance if that's the case

**Comment by pmdevita** [comment]
It would be nice to have per app functionality like Django, I have an application that needs to load apps modularly and having all of the migrations get merged into one file means I can't properly separate them.

**Comment by the-akpan** [comment]
Has there been any update on this, because I am having the same issue.

**Comment by the-akpan** [comment]
I made a slight hack to work around this, wrote a script to loop through the `apps` keys of the config dict then ran the aerich command on each individually.
```python
errors = []
for app in apps:
    if os.system(f"aerich --app {app} {command}"):
        errors.append(app)
if errors:
    print("Errors encountered on %" % ",".join(errors))
```

**Comment by long2ice** [comment]
If not `--app` specific, maybe we should migrate all app default

**Comment by the-akpan** [comment]
> If not `--app` specific, maybe we should migrate all app default

That would be the expected behavior.

**Comment by long2ice** [comment]
PR is welcome!

**Comment by intigratech** [comment]
@abondar 
Is this something that someone is working on at the moment or is at least planned for a future release?

**Comment by waketzheng** [comment]
@intigratech No plan yet. May consider it after #366

**Comment by Kokleng-Dev** [comment]
Why everyone ignore this ?
please consider do it like django orm does.

the benefit 1folder 1migration is when we develop it as modular and seperate it later for micro service. 
it is the benefit of django architecture. 

Anyone, could help me to implement it like django orm migration? 

i am using fastapi and very interest it in toroise orm.
